apps/frontend/src/app/settings/settings/settings.component.ts

```diff
@@ -130,8 +130,17 @@ export class SettingsComponent implements OnInit {
       return;
     }
     this.saveUser().subscribe({
-      next: () => this.toastService.success('Account Settings', 'Successfully saved account settings.'),
-      error: error => this.toastService.error('Account Settings', 'Failed to save account settings.', error),
+      next: () =>
+        this.toastService.success(
```

Comment:
Strange formatting that suggests multiple statements are possible whereas its only one. Either remove line break or add braces